xml/Microsoft.Build.Tasks/DownloadFile.xml

```diff
@@ -170,8 +170,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>This method is called by the build engine to begin task execution. A task uses the return value to indicate whether it was successful. If a task throws an exception out of this method, the engine will automatically assume that the task has failed.</summary>
+        <returns>true, if successful</returns>
```

Comment:
```suggestion
        <summary>Called by the build engine to begin task execution. A task uses the return value to indicate whether it was successful. If a task throws an exception out of this method, the engine will automatically assume that the task has failed.</summary>
        <returns><see langword="true"/> if successful; <see langword="false"/> otherwise.</returns>
```